Game.java

```diff
@@ -7,10 +7,11 @@
 
     private InputStream stream;
     private Board board;
-    private String gameMode = "pvp";
+    public String gameMode = "pvp";
     private Player playerActive;
     private Player playerDeactive;
     private BufferedReader inputReader;
+    private ConsoleGame consoleGame = new ConsoleGame(this);
 
     public Game(InputStream stream, Board board) {
```

Comment:
Game shouldn't know about InputStream after ConsoleGame takes on this responsibility.